### DISCUSSION TITLE
Error with backup during installation

### DISCUSSION THREAD

**Discussion by jbaughma** [mpurses/Sonder#36] [Q&A]
I must be missing something obvious, but I run into this error on installing the new version:

![image](https://user-images.githubusercontent.com/98043922/150174141-fabd8003-a975-439f-9100-b5aeff4006b1.png)

The current version is 2.6.0 and I recently updated Rainmeter to 4.5.8 and it's on a Win10 PC. Any ideas? Thanks!

**Comment by mpurses** [comment]
I've encountered this randomly as well. I've never been able to figure out why/how it happens to tell the Rainmeter devs. When debugging it seems like it fails to get write access at some point but doesn't give me more info on why. 

You will lose your saved color/size changes, but you can uncheck Backup skins under the Advanced dropdown. That should hopefully let you upgrade.

![image](https://user-images.githubusercontent.com/642218/150179438-90040716-7904-48c0-a1c6-627c6041114c.png)

**Reply by jbaughma** [reply]
Thanks for the quick reply! Yeah, I checked the write permission on the backup folder and that didn't seem to be an issue so I wasn't sure where it was hitting this snag. I'll take your suggestion, the color/size settings won't be hard to recreate for my setup.
